Sources/SwiftSyntax/gyb_generated/syntax_nodes/SyntaxPatternNodes.swift

```diff
@@ -104,13 +102,51 @@ public struct EnumCasePatternSyntax: PatternSyntaxProtocol, SyntaxHashable {
       unexpectedAfterAssociatedTuple?.raw,
     ]
     let data: SyntaxData = withExtendedLifetime(SyntaxArena()) { arena in
-      let raw = RawSyntax.makeLayout(kind: SyntaxKind.enumCasePattern,
-        from: layout, arena: arena)
+      let raw = RawSyntax.makeLayout(
+        kind: SyntaxKind.enumCasePattern, from: layout, arena: arena,
+        leadingTrivia: leadingTrivia, trailingTrivia: trailingTrivia)
       return SyntaxData.forRoot(raw)
     }
     self.init(data)
   }
 
+  /// This initializer exists solely because Swift 5.6 does not support
+  /// `Optional<ConcreteType>.none` as a default value of a generic parameter.
+  /// The above initializer thus defaults to `nil` instead, but that means it
+  /// is not actually callable when either not passing the defaulted parameter,
+  /// or passing `nil`.
+  ///
+  /// Hack around that limitation using this initializer, which takes a
+  /// `Missing*` syntax node instead. `Missing*` is used over the base type as
+  /// the base type would allow implicit conversion from a string literal,
+  /// which the above initializer doesn't support.
+  public init(
+    leadingTrivia: Trivia? = nil,
+    _ unexpectedBeforeType: UnexpectedNodesSyntax? = nil,
+    type: MissingTypeSyntax? = nil,
+    _ unexpectedBetweenTypeAndPeriod: UnexpectedNodesSyntax? = nil,
+    period: TokenSyntax = .periodToken(),
+    _ unexpectedBetweenPeriodAndCaseName: UnexpectedNodesSyntax? = nil,
+    caseName: TokenSyntax,
+    _ unexpectedBetweenCaseNameAndAssociatedTuple: UnexpectedNodesSyntax? = nil,
+    associatedTuple: TuplePatternSyntax? = nil,
+    _ unexpectedAfterAssociatedTuple: UnexpectedNodesSyntax? = nil,
+    trailingTrivia: Trivia? = nil
+  ) {
+    self.init(
+      leadingTrivia: leadingTrivia,
+      unexpectedBeforeType,
+      type: Optional<TypeSyntax>.none,
```

Comment:
You are dropping `unexpectedBetweenTypeAndPeriod` here. AFAICT you should remove `skip_next` in `SyntaxNodes.swift.gyb.template`.

Reply:
Argh. Thanks. I thought I removed all that but apparently not.